lab_3_generate_by_ngrams/main.py

```diff
@@ -208,6 +325,25 @@ def build(self) -> int:
         In case of corrupt input arguments or methods used return None,
         1 is returned
         """
+        if not isinstance(self._encoded_corpus, tuple) or len(self._encoded_corpus) == 0:
```

Comment:
`== 0` 😠 

please check all the similar places for this thing, it is a bad practice, and your code is way too good to use it